extensions/binding/org.eclipse.smarthome.binding.weatherunderground/src/main/java/org/eclipse/smarthome/binding/weatherunderground/handler/WeatherUndergroundHandler.java

```diff
@@ -298,6 +298,9 @@ private WeatherUndergroundJsonData getWeatherData(Set<String> features, String l
             } else if (result.getResponse() == null) {
                 errorDetail = "missing response sub-object";
             } else if (result.getResponse().getErrorDescription() != null) {
+                if ("keynotfound".equals(result.getResponse().getErrorType())) {
```

Comment:
How is this related to this feature?
Note that if the api key is a mandatory config parameter, the framework will make sure that the handler isn't even started, if the key is not set. So I would claim that this is not related, right?